lampetia-security-model/src/main/scala/lampetia/security/model/models.scala

```diff
@@ -279,7 +298,7 @@ trait SecurityModel {
       }
       object parentResource extends Composite[Option[Resource]] {
         val resourceId =
-          property[Option[ResourceId]]("resourceId")
```

Comment:
OK.
please add a test case to make sure parentResource can be inserted as None.

On 8/7/15 6:38 PM, rosama86 wrote:
>
> In 
> lampetia-security-model/src/main/scala/lampetia/security/model/models.scala 
> <https://github.com/hkarim/lampetia/pull/2#discussion_r36536439>:
>
> > @@ -279,7 +298,7 @@ trait SecurityModel {
> >        }
> >        object parentResource extends Composite[Option[Resource]] {
> >          val resourceId =
> > -          property[Option[ResourceId]]("resourceId")
>
> since parent resource is optional, resourceId don't need to be 
> optional too. right?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/hkarim/lampetia/pull/2/files#r36536439>.
>